pkg/features/features.go

```diff
@@ -131,6 +131,12 @@ const (
 	// owner: @dimityrmirchev
 	// alpha: v1.54.0
 	DefaultSeccompProfile featuregate.Feature = "DefaultSeccompProfile"
+
+	// CoreDNSQueryRewriting enables automatic DNS query rewriting in shoot cluster's CoreDNS to shortcut name resolution of
+	// fully qualified (in-cluster) names and (out-of-cluster) names, which follow a user-defined pattern.
```

Comment:
Can it be written like

```suggestion
	// fully qualified in-cluster and out-of-cluster names, which follow a user-defined pattern.
```

Reply:
Addressed with latest commit.